test/snowpack/build/build-import-proxy.test.js

```diff
@@ -0,0 +1,80 @@
+const {wrapImportProxy} = require('../../../snowpack/lib/build/build-import-proxy.js');
+
+describe('build-import-proxy', () => {
```

Comment:
An example test that tests code generation for import transformation. Note that in this folder, the test filename matches the original Snowpack file.